app/src/main/java/com/babylon/wallet/android/domain/usecases/signing/SignTransactionUseCase.kt

```diff
@@ -66,23 +69,21 @@ class SignTransactionUseCase @Inject constructor(
     }
 
     data class Request(
-        private val manifest: TransactionManifestData,
+        val manifestData: TransactionManifestData,
```

Comment:
also here you might pass just the sargon's `Message` instead of `TransactionManifestData`?

Reply:
Besides `Message` we also need the `NetworkId` and the `TransactionManifest`, so the `TransactionManifestData` is used for convenience because it contains all of these. I think this suggestion is related also to the other comment about the purpose and the usage of the `TransactionManifestData`, we can definitely improve this, maybe during the work at adding support for sub-intents?